packages/@sanity/dashboard/src/widgets/project-users/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import sanityClient from 'part:@sanity/base/client'
+import Spinner from 'part:@sanity/components/loading/spinner'
+import styles from './index.css'
+import DefaultPreview from 'part:@sanity/components/previews/default'
+import {List, Item} from 'part:@sanity/components/lists/default'
+import AnchorButton from 'part:@sanity/components/buttons/anchor'
+import ToolIcon from 'react-icons/lib/go/tools'
+
+function getInviteUrl(projectId) {
+  return `https://manage.sanity.io/projects/${projectId}/team/invite`
+}
+
+class ProjectUsers extends React.Component {
+  static propTypes = {}
+  static defaultProps = {}
+
+  state = {
+    project: null,
+    users: null,
+    error: null
+  }
+
+  sortUsersByRobotStatus = (userA, userB) => {
+    const {members} = this.state.project
+    const membershipA = members.find(member => member.id === userA.id)
+    const membershipB = members.find(member => member.id === userB.id)
+    if (membershipA.isRobot) {
+      return 1
+    }
+    if (membershipB.isRobot) {
+      return -1
+    }
+    return 0
+  }
+
+  componentDidMount = () => {
+    const {projectId} = sanityClient.clientConfig
```

Comment:
Prefer `sanityClient.config()` over `sanityClient.clientConfig` (the latter is not documented and should be considered internals)